config/application.rb

```diff
@@ -45,5 +45,7 @@ class Application < Rails::Application
 
     # Compress the responses to reduce the size of html/json controller responses.
     config.middleware.use Rack::Deflater
+
+    config.hosts << ENV["DEPLOYED_HOST"]
```

Comment:
It's the first time I see this config 👀 
Could you share more about why we need this change? :) 

Thanks

Reply:
<img width="1137" alt="Screen Shot 2023-06-02 at 11 55 43" src="https://github.com/blyscuit/rails-ic/assets/6356137/53c82693-d749-4b83-8ef2-dc25b92d3643">
I got this error running the app straight from template so I followed the instruction and it works fine. 🙏

Reply:
Thanks, all clear :)
It is interesting as in our current client project, we didn't need it at all. Maybe because the infrastructure is different 🤔 

https://guides.rubyonrails.org/configuring.html#actiondispatch-hostauthorization